packages/devtools/src/composite.ts

```diff
@@ -17,6 +19,12 @@ import { createRuntimeDefinition } from './formats/runtime.js'
 import { createAbstractCompositeDefinition } from './schema/compiler.js'
 import type { AbstractModelDefinition } from './schema/types.js'
 
+// The hardcoded "model" StreamID that all Model streams have in their metadata. This provides
+// a "model" StreamID that can be indexed to query the set of all published Models.
+// The StreamID uses the "UNLOADABLE" StreamType, and has string representation: "kh4q0ozorrgaq2mezktnrmdwleo1d"
+
+const MODEL_STREAM_ID = 'kh4q0ozorrgaq2mezktnrmdwleo1d'
```

Comment:
```suggestion
const MODEL_STREAM_ID = Model.MODEL.toString()
```
Better use the stream ID defined on the model itself than copying it I think.